### ISSUE TITLE
Unexpected AST node: QSTRING with CREATE statement

### ISSUE THREAD

**Issue by movedoa** [Riverside-Software/sonar-openedge#310]
Happens with 2.0.3

```
CLASS test:

   METHOD PUBLIC VOID Foo():
      DEFINE VARIABLE lhObjShell AS COMPONENT-HANDLE NO-UNDO.
      CREATE "WScript.Shell":U lhObjShell.
   END METHOD.

END CLASS.
```

**Comment by gquerret** [comment]
Looks like anything can follow the `CREATE` statement :-(
I'll relax once again the rule !

**Comment by movedoa** [comment]
The most ambiguous keyword of all time 😄

**Comment by gquerret** [comment]
Fixed in master

**Comment by maelstraggiotti** [comment]
Is it supposed to be fixed in 2.0.4 snapshot ? Statements like ```CREATE "PROX.ControlDef":U hHandle NO-ERROR.``` still fails.

**Comment by gquerret** [comment]
@maelstraggiotti  Yes, it is fixed in sonar-openedge-2.0.4-SNAPSHOT ( not in the rules).
